src/main/java/org/example/annotations/Column.java

```diff
@@ -0,0 +1,30 @@
+package src.main.java.org.example.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.METHOD, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Column {
+    String name() default "";
+
+    boolean unique() default false;
+
+    boolean nullable() default true;
+
+    boolean insertable() default true;
+
+    boolean updatable() default true;
+
+    String columnDefinition() default "";
+
+    String table() default "";
+
+    int length() default 255;
+
+    int precision() default 0;
+
+    int scale() default 0;
+}
```

Comment:
I don't know that we will need all of these members but either way I'm fine by it.

Reply:
Yup, on it